app/assets/javascripts/application.js.erb

```diff
@@ -44,6 +44,7 @@
 //= require protocols/header
 //= require marvinjslauncher
 //= require_directory ./repositories/renderers
+//= require_directory ./repositories/renderers/columns
```

Comment:
Is it needed here?

Reply:
Yes or we can use `require_tree` for `.repositories/renderers`